src/Exception/NoDriverException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Exception;
+
+/**
+ * @author Andrey F. Mindubaev <covex.mobile@gmail.com>
+ */
+final class NoDriverException extends \RuntimeException
+{
+    const DEFAULT_MESSAGE = 'The child node "db_driver" at path "sonata_media" must be configured.';
```

Comment:
Why put this in a constant? It's public now, and it's only ever used once.

Reply:
Yeah but why not do this: https://3v4l.org/WmvS8 or this: https://3v4l.org/ROnrq?

Reply:
Just i did not think, that it is important. Next time i will try not to create new not necessary entities